### ISSUE TITLE
Unpredictable behavior on deselecting the cursor

### ISSUE THREAD

**Issue by javidh** [microsoft/vscode#85962]
- VSCode Version: 1.40.2
- OS Version: maOS 10.4.06 (Mojahve)

Steps to Reproduce:

1. Try to add a few cursors (option + click)
2. Try to remove one by (option + click)

Sometimes it works as expected sometimes it selects the text between the cursor and the previous one. 

Does this issue occur when all extensions are disabled?: Yes

**Comment by alexandrudima** [comment]
@javidh I was not able to reproduce what you describe. When I do option+click on a cursor or on a selection, it gets removed as I would expect:

![TO_UPLOAD](https://user-images.githubusercontent.com/5047891/70140679-8ffecc80-1695-11ea-883b-3fde5c52e1b6.gif)

**Comment by javidh** [comment]
![vs-code-bug](https://user-images.githubusercontent.com/6296082/70205605-8ce5f980-16f2-11ea-9b31-1895e8a54bc4.gif)

What info can I provide you with?

**Comment by javidh** [comment]
As it is seen in the gif, it happens randomly.

**Comment by alexandrudima** [comment]
@javidh Are you using `window.zoomLevel`?

**Comment by javidh** [comment]
@alexandrudima Yes I do. This is my setttings.json:

```
{
  "files.autoSave": "afterDelay",
  "ruby.useBundler": false, //run non-lint commands with bundle exec
  "ruby.useLanguageServer": true, // use the internal language server (see below)
  "ruby.lint": {
    "rubocop": {
      "useBundler": true // enable rubocop via bundler
    },
    "reek": {
      "useBundler": true // enable reek via bundler
    }
  },
  "ruby.format": "rubocop",
  "workbench.editor.showTabs": true,
  "diffEditor.renderSideBySide": true,
  "search.location": "sidebar", // use rubocop for formatting
  "terminal.integrated.fontFamily": "hack",
  "window.zoomLevel": 0,
  "gitlens.views.repositories.files.layout": "list"
}
```

**Comment by alexandrudima** [comment]
I think this shares the same root cause as #83150. The textarea is rendered at the location of the primary cursor and sometimes, especially on some resolutions/zoom combinations it is possible to hit the textarea. I have just pushed a fix for this in #83150

Tomorrow's insiders build will contain the fix.
